src/widget/form/settings/avform.cpp

```diff
@@ -228,9 +228,15 @@ void AVForm::updateVideoModes(int curIndex)
     unsigned short prefFPS = Settings::getInstance().getCamVideoFPS();
     // Iterate backwards to show higest resolution first.
     videoModes.clear();
+    QVector<int> usedInds;
     for(auto iter = bestModeInds.rbegin(); iter != bestModeInds.rend(); ++iter)
     {
         int i = iter->second;
+        // don't use the same mode twice
+        if (usedInds.contains(i))
```

Comment:
uhm, why introduce a new variable here and not just do `videoMode.contains(mode)` ?